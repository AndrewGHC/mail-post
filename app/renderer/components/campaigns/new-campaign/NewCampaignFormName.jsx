import React from 'react';
import { reduxForm, Field } from 'redux-form';
import { TextField } from 'redux-form-material-ui';

import { FORM_NEW_CAMPAIGN } from '../../../constants';

const NewCampaignFormName = () => <Field name="username" component={TextField} placeholder="Street" />;

export default reduxForm({
  form: FORM_NEW_CAMPAIGN,
})(NewCampaignFormName);
