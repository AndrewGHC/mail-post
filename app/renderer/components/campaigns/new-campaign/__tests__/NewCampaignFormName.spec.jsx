import React from 'react';
import { shallow } from 'enzyme';

import NewCampaignFormName from '../NewCampaignFormName';

describe('NewCampaignFormName', () => {
  it('matches snapshot', async () => {
    const wrapper = shallow(<NewCampaignFormName />);
    expect(wrapper).toMatchSnapshot();
  });
});
