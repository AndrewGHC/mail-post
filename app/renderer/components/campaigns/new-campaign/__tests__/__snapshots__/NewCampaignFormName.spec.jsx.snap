// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewCampaignFormName matches snapshot 1`] = `
<Connect(Form(NewCampaignFormName))
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="FORM_NEW_CAMPAIGN"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldError={[Function]}
  shouldValidate={[Function]}
  shouldWarn={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
  updateUnregisteredFields={false}
/>
`;
